demisto_sdk/commands/format/format_module.py

```diff
@@ -86,8 +86,9 @@ def format_manager(input: str = None,
     Returns:
         int 0 in case of success 1 otherwise
     """
-    if input:
-        files = get_files_in_dir(input, ['json', 'yml', 'py', 'md'])
+    if input != 'None':
+        if isinstance(input, str):
```

Comment:
i think it is better to keep this code as it was and fix the function call in `__main__.py`:
```python
format_manager(
    input=str(input) if input else None,
    ...
)
```

@roysagi WDYT?